include/dpp/discordvoiceclient.h

```diff
@@ -48,22 +48,47 @@
 #include <future>
 #include <functional>
 #include <chrono>
+#include <set>
 
 struct OpusDecoder;
 struct OpusEncoder;
 struct OpusRepacketizer;
 
 namespace dpp {
 
+/**
+ * @brief Sample rate for OPUS (48khz)
+ */
+[[maybe_unused]] inline constexpr int32_t opus_sample_rate_hz = 48000;
+
+/**
+ * @brief Channel count for OPUS (stereo)
+ */
+[[maybe_unused]] inline constexpr int32_t opus_channel_count = 2;
```

Comment:
this is passed to an opus function that takes an int32_t, plus used in a bunch of 32 bit calculations. Probably best to keep it 32 bits to avoid complications for the sake of saving one byte...

Reply:
Makes sense!